cli_tools/diagnostics/main.go

```diff
@@ -176,6 +176,6 @@ func main() {
 	os.RemoveAll(tmpFolder)
 
 	if nonFatalErrorsPresent {
-		log.Fatal("Errors occured while collecting and zipping some logs.\nUnaffected logs were still packaged and available.")
+		log.Print("Errors occured while collecting and zipping some logs.\nUnaffected logs were still packaged and available.")
```

Comment:
why we change to print? can we still see it from serial log?

Reply:
The tests are using WinRM to trigger this tool, if some trivial errors happened, Fatal returned a panic. So the tests stopped as they got stderr.
Yes, we can still see it from serial log.

Reply:
but this is really some kind of error we should capture during our test

Reply:
Well the existing code doesn't make much sense in the first place: if _non fatal_ errors present, log _fatal_?

I believe the errors that are being referred to here are due to the tool failing to gather docker/kubernetes logs, which are not guaranteed to be present. We should distinguish between failures like that (which can be logged as just warnings/info) and other failures that should be error/fatal level. Yang, how hard would it be to do that?

Reply:
Right, the errors happens if kubernetes logs("C:\etc\kubernetes\logs") or crash dump("C:\Windows\MEMORY.dmp") doesn't exist. 
It's not difficult, we can log warnings/info immediately inside the kubernetes logs collector.

Reply:
agree. that will be a better approach

Reply:
Ok, updated in the PR

Reply:
thx. it is mush better. please let me know when the fix is released.